login-workflow/docs/routing.md

```diff
@@ -8,7 +8,80 @@ Because this workflow package is router-agnostic, you will be required to set up
 
 You will also want to set up Auth/Guest Guard wrappers to control which users can access which screens / routes. For more information see [Protecting Routes](#protecting-routes) below.
 
-### Authentication
+### Authentication (Okta Redirect)
+
+The **Authentication** workflow screens are rendered individually on separate routes (e.g., the Okta Redirect Login screen is on '/login'). This means you can deep-link to any of these screens directly if you have them configured.
+
+You have to add a `Security` component as a wrapper to the `Routes` from [@okta/okta-react](https://www.npmjs.com/package/@okta/okta-react) package. This component initializes the Okta authentication context and provides methods to interact with Okta.
+
+For more information on the `OktaAuthContextProvider` , refer to the [Authentication Workflow](./authentication-workflow.md) Guide.
```

Comment:
This reads weirdly — you say "For **more** information about the OktaAuthContextProvider, check out this link", but you never mentioned the OktaAuthContextProvider in the first place.